config/clients.yaml

```diff
@@ -0,0 +1,6 @@
+client-name:
+  client_id: client_id
+  client_secret: client_secret
+  redirect_uri: http://www.example.com/check
+
+# ...
```

Comment:
The comment was in the original file to demonstrate repetition, not needed here.